src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -440,12 +440,15 @@ public static void cloneFromScm(
         PluginSourcesUnavailableException lastException = null;
         for (String connectionURL : connectionURLs) {
             if (connectionURL != null) {
+                if (StringUtils.startsWith(connectionURL, "scm:git:")) {
```

Comment:
code was moved up here so that `cloneImpl` only has to deal with real `git` URLs